schemas/com.mandrill/message_clicked/jsonschema/1-0-2

```diff
@@ -0,0 +1,290 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for a Mandrill message clicked event",
+	"self": {
+		"vendor": "com.mandrill",
+		"name": "message_clicked",
+		"format": "jsonschema",
+		"version": "1-0-2"
+	},
+
+	"type": "object",
+	"properties": {
+		"_id": {
+			"type": "string",
+			"description": "The unique identifier of the message that generated the event. This is not an event identifier, but rather a reference to the message ID for the email that was sent, opened, clicked, etc."
+		},
+		"ip": {
+			"type": "string",
+			"description": "IP address where the event originated"
+		},
+		"location": {
+			"type": ["object", "null"],
+			"description": "The geolocation where the event occurred. Value will be null if no location can be determined from the IP address of the event.",
+			"properties": {
+				"city": {
+					"type": "string"
+				},
+				"country_short": {
+					"type": "string",
```

Comment:
If `country_long` is nullable should country short be nullable as well?

Reply:
This is a question I was pondering a lot when updating the schemas – almost all of the properties in the previous schema version were non-required but non-nullable. I was thinking of making them all non-required but nullable. In the end, I went for minimal changes to the schema since these are just patch updates. I'm happy to make it nullable, but in that case, should we change everything to nullable to be consistent?